README.md

```diff
@@ -30,6 +30,10 @@ This SDK version is compatible with the Stark Infra API v2.
         - [Stock](#query-issuingstocks): View your current stock of a certain IssuingDesign linked to an Embosser on the workspace
         - [Restock](#create-issuingrestocks): Create restock orders of a specific IssuingStock object
         - [EmbossingRequest](#create-issuingembossingrequests): Create embossing requests
+        - [TokenRequest](#create-an-issuingtokenrequest): Create token requests
+        - [TokenAuthorization](#process-issuingtokenauthorizations): process token authorizations
```

Comment:
- [Token](#process-issuingtokenauthorizations): Authorize and manage your tokens

Reply:
alinhar